mrp_bom_attribute_match/models/mrp_bom.py

```diff
@@ -29,6 +31,18 @@ class MrpBomLine(models.Model):
         compute="_compute_product_uom_category_id",
     )
 
+    @api.constrains("component_template_id", "parent_product_tmpl_id")
```

Comment:
```suggestion
    @api.constrains("component_template_id")
```

To supress this log warning:

```
2023-06-06 07:20:16,529 250 WARNING odoo odoo.models: method mrp.bom.line._check_component_template_recursion: @constrains parameter 'parent_product_tmpl_id' is not writeable 
```